cluster/network.go

```diff
@@ -284,9 +340,98 @@ func (c *Cluster) doWeaveDeploy(ctx context.Context, data map[string]interface{}
 	return c.doAddonDeploy(ctx, pluginYaml, NetworkPluginResourceName, true)
 }
 
+func (c *Cluster) doAciDeploy(ctx context.Context, data map[string]interface{}) error {
+	refreshTime, err := atoiWithDefault(c.Network.AciNetworkProvider.ApicRefreshTime, DefaultApicRefreshTime)
```

Comment:
What is the reason this (and the one below) is not in `cluster/defaults.go`? This currently panics when not provided. I think the whole part for defaults and validation should go into `cluster/defaults.go` and `cluster/validation.go`.

Reply:
Thanks for getting back. I tried this just now that you mentioned, It doesn't panic when apic_refresh_time is not given. This line is there for that exact reason. I edited the cluster.yaml I am using now exactly in the post above.
I have validated the options that are required for ACI and these don't apply to other CNIs which don't have the physical fabric involved. Keeping them common will not help other CNIs. We use the other common network options like MTU and pod subnet that are required by all CNIs. We can discuss more about this offline over email if you prefer

Reply:
I ran this PR with just specifying the network plugin `ACI` and nothing else and it panics on this line.

The problem here is that if any of these defaults error out, the cluster is provisioned but the network provisioning errors out while this can be easily caught by defaults/validation.

Reply:
Ok. For ACI, not all arguments have defaults and defaults cannot be provided for all arguments because they pertain to specific values like the address of the ACI controller, credentials for it, other subnets. Running without any arguments is bound to fail althought not at this argument. We have a tool called acc-provision that generates this portion of cluster.yml, so customers dont have to do this by hand. https://github.com/noironetworks/acc-provision.

Reply:
Ok.. I will move these to cluster/defaults.go as you mentioned and submit a PR shortly.